Jellyfin.Server.Implementations/Devices/DeviceManager.cs

```diff
@@ -78,23 +99,19 @@ await using (dbContext.ConfigureAwait(false))
                 dbContext.Devices.Add(device);
 
                 await dbContext.SaveChangesAsync().ConfigureAwait(false);
+                var newDevice = await dbContext.Devices
```

Comment:
Why is it being retrieved here? `device` is already tracked by EF

Reply:
This is leftover code. We were caching user in device cache as well and the device passed in will not have a user attached to it. But now we are querying user directly from the user cache so this can be safely removed and add the device to the cache directly